azavea/raster-foundry
# Feature/lol/create layer selector component

Moves hard coded values from the Library component into sub components wired to backbone models and collections.

Connects #53 

To test:
* Bundle and test out the layer selector.
* Add a break point after the rendering of the library and update any of the layer models.
* Ensure the component rerenders.
* Click component UI and check to see that dummy handlers are firing.
